t/buy-sig-libp2p-local-integration-test.ss

```diff
@@ -0,0 +1,219 @@
+         (lambda ()
+           (answer-questions
+             [["Choose your identity:"
+               (lambda (id) (string-prefix? "t/bob " id))]])))
+
+      ;; Start buyer
+       (set! proc-buyer
+         (open-process
+          [path: "./glow"
+           arguments: ["start-interaction"
+                       "--glow-path" (source-path "dapps")
+                       "--evm-network" "pet"
+                       "--test"
+                       ;; "--handshake" "nc -l 3232"
+                       ;; For the sake of testing both the cli flag and the
+                       ;; console prompt, we supply one parameter here and the
+                       ;; other below.
+                       "--params" (string-append "{\"price\": " (number->string price) "}")
+
+                       "--host-address" "/ip4/10.0.0.6/tcp/10300"
+                       "--off-chain-channel" "libp2p"
+                       ;; TODO: derive the peerid
+                       "--dest-address" "/ip4/10.0.0.6/tcp/10333/ipfs/16Uiu2HAmUXHHL7qEMNmwgynPF3GLGjo8n72TDnMPAgAFYPmnfpv8"
+
+                       ;; Similarly, specify one of the participants here. There are only two,
+                       ;; so this test doesn't excercise the logic to read this from stdin,
+                       ;; but the other integration tests do.
+                       ;;
+                       ;; N.B. this is bob's id.
+                       "--participants" "{\"Seller\": \"0xb0bb1ed229f5Ed588495AC9739eD1555f5c3aabD\"}"
+                       "--assets" "{\"DefaultToken\": \"PET\"}"
+                       ]
+           ]))
+
+       (displayln "Filling up buyer prompt")
+
+       ;; Fill up buyer prompt
+       ;; (def peer-command
+         (with-io-port proc-buyer
+           (lambda ()
+             (answer-questions
+              [["Choose your identity:"
+                (lambda (id)
+                  (string-prefix? "t/alice " id))]
+               ["Choose application:"
+                "buy_sig"]
+               ["Choose your role:"
+                "Buyer"]])
+             (supply-parameters
+              [["digest" (string-append "0x" (hex-encode digest))]])
+             (set-initial-block 1000) ; Provides an offset from the current-block,
+                                      ; so we have ample time (in blocks) to create a contract
+                                      ; and for other active participants to run side
+                                      ; of the interaction before timeout.
+                                      ;
+                                      ; Also used for regression testing against:
+                                      ; https://gitlab.com/mukn/glow/-/issues/195
+             ))
+
+
+
+       (displayln "Choosing role")
+
+       (with-io-port proc-seller
+         (lambda ()
+           ;; Wait for handshake to come in
+           (thread-sleep! 10)
+           ;; (displayln "2")
+           ;; (force-output)
```

Comment:
Should not leave code commented out like this.

Reply:
Removed